configuration.php

```diff
@@ -185,3 +185,4 @@
 $this->provideCssFile('icons.less');
 $this->provideCssFile('lists.less');
 $this->provideCssFile('widgets.less');
+$this->provideCssFile('condition-list.less');
```

Comment:
The `provideCssFile` calls are alphabetically ordered. Please adjust accordingly.